randovania/games/samus_returns/json_data/header.txt

```diff
@@ -77,6 +87,14 @@ Templates
           Ice Beam
           Charge Beam or Freeze Enemy with Melee (Intermediate)
 
+* Shoot Super Missile:
+      Super Missile and Super Missile Launcher
```

Comment:
This should not have a super launcher listed

Reply:
Same as with pbs